### DISCUSSION TITLE
跨盘复制文件有问题，它是直接生成一个同名的文件夹而不是复制文件

### DISCUSSION THREAD

**Discussion by creeper2020** [AlistGo/alist#1670] [General]
ERRO[2022-09-14 09:33:49] error [object not found
github.com/alist-org/alist/v3/internal/op.Get
	/home/runner/work/alist/alist/internal/op/fs.go:127
github.com/alist-org/alist/v3/internal/op.Put
	/home/runner/work/alist/alist/internal/op/fs.go:300
github.com/alist-org/alist/v3/internal/fs.copyFileBetween2Storages
	/home/runner/work/alist/alist/internal/fs/copy.go:95
github.com/alist-org/alist/v3/internal/fs.copyBetween2Storages.func2
	/home/runner/work/alist/alist/internal/fs/copy.go:75
github.com/alist-org/alist/v3/pkg/task.(*Task[...]).run
	/home/runner/work/alist/alist/pkg/task/task.go:75
github.com/alist-org/alist/v3/pkg/task.(*Manager[...]).do.func1
	/home/runner/work/alist/alist/pkg/task/manager.go:33
runtime.goexit
	/opt/hostedtoolcache/go/1.18.5/x64/src/runtime/asm_amd64.s:1571
failed to get dir [/我的应用收藏/飞蛾扑火.flac]] while run task [copy [/pan](/pan/music/飞蛾扑火.flac) to [/mobile](/我的应用收藏/飞蛾扑火.flac)]

**Comment by lxl8800** [comment]
一样遇到了这个问题
